src/pages/Sign/Login/Login.js

```diff
@@ -1,8 +1,79 @@
 import React, { Component } from "react";
 import "./Login.scss";
+import { Link } from "react-router-dom";
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      emailVal: "",
+      pwVal: "",
+    };
+  }
+
+  handleEmailInput = e => {
+    this.setState({
+      emailVal: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pwVal: e.target.value,
+    });
+  };
+
+  checkValid = () => {
+    const { emailVal, pwVal } = this.state;
+
+    return emailVal.includes("@") && pwVal.length >= 6;
+  };
+
+  handleLogin = () => {
+    this.checkValid() && this.props.history.push("/login");
+  };
 
-export default class Login extends Component {
   render() {
-    return <div>Hi</div>;
+    const { emailVal, pwVal } = this.state;
+    return (
+      // 모달창 만들고 그 안에 아래 코드는 컴포넌트로 만들어서 삽입
+      <div className="loginContainer">
+        <h1>로그인</h1>
+        {/* h1 자리에 맛남츄 로고를 넣고 싶어요 */}
+        <input
+          type="text"
+          placeholder="이메일"
+          className="inputEmail"
+          onChange={this.handleEmailInput}
+        />
+        {emailVal && !emailVal.includes("@") && (
+          <div className="warning">
+            이메일 형식에 맞게 입력해 주세요. ex '@' 포함
+          </div>
+        )}
+        <input
+          type="password"
+          placeholder="비밀번호"
+          className="inputPw"
+          onChange={this.handlePwInput}
+        />
+        {pwVal && pwVal.length < 6 && (
+          <div className="warning">6글자 이상 입력해 주세요.</div>
+        )}
+        <button
+          className={this.checkValid() ? "loginBtn" : "loginBtn disabled"}
```

Comment:
다음과 같이 template literal을 이용해 작성할 수도 있습니다!

```suggestion
          className={`loginBtn ${this.checkValid() ? '' : 'disabled'}`}
```

Reply:
네 감사합니다! 다음엔 이렇게 써보겠습니다!